Dockerfile

```diff
@@ -11,16 +11,19 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 #
-FROM openzipkin/zipkin:1.23.3
+FROM openzipkin/zipkin:1.25.0
 MAINTAINER OpenZipkin "http://zipkin.io/"
 
 ENV ZIPKIN_AWS_REPO https://jcenter.bintray.com
-ENV ZIPKIN_AWS_VERSION 0.2.5
+ENV ZIPKIN_AWS_VERSION 0.3.1
 
 RUN apk add unzip && \ 
   curl -SL $ZIPKIN_AWS_REPO/io/zipkin/aws/zipkin-autoconfigure-collector-sqs/$ZIPKIN_AWS_VERSION/zipkin-autoconfigure-collector-sqs-$ZIPKIN_AWS_VERSION-module.jar > sqs.jar && \
+  curl -SL $ZIPKIN_AWS_REPO/io/zipkin/aws/zipkin-autoconfigure-collector-kinesis/$ZIPKIN_AWS_VERSION/zipkin-autoconfigure-collector-kinesis-$ZIPKIN_AWS_VERSION-module.jar > kinesis.jar && \
   unzip sqs.jar -d sqs && \
+  unzip kinesis.jar -d kinesis && \
   rm sqs.jar && \
+  rm kinesis.jar && \
```

Comment:
Thinking about abstracting these 3 steps to a function but don't think its worth the trouble unless we got a 3rd module